mountpoint-s3-client/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
   ([awslabs/aws-c-s3#516](https://github.com/awslabs/aws-c-s3/pull/516))
 * The memory limit for CRT Client can now be configured with the `S3ClientConfig::memory_limit_in_bytes` method.
   ([#1363](https://github.com/awslabs/mountpoint-s3/pull/1363))
+* Fix missing credential caching when using `S3ClientAuthConfig::Profile` credential configuration.
```

Comment:
nit: it's my pet peeve, so feel free to ignore, but do we need the "Other changes" header if these are the only changes?

Reply:
I'll ignore for this PR. I prefer keeping it, maybe we can keep the new features header and say none or rename this to "Small changes" or something